### DISCUSSION TITLE
migrating Github education perks to a new github account

### DISCUSSION THREAD

**Discussion by Siyanda-m2** [community/community#133108] [GitHub Education] [answered]
### Hi everyone,

I recently unlinked my university provided email account from the GitHub account I was using due to being locked out of it. (2FA phone and laptop with the access codes file were stolen). I have since created a new GitHub account and linked the student email to it. is there a way to migrate the GitHub benefits to this account if I can prove I'm still the same user? i tried reapplying only to get this message "myemail@myschool has already been used to obtain an academic discount on GitHub account(s) old_username."

**Comment by davevad93** [comment, marked as answer]
Hi @Siyanda-m2 ,

It's not possible to transfer Student benefits from one account to another. You can merge two accounts although the account with the benefits would still be the main one. 

For more reference on the topic here's the official response from a member of the GitHub Education team . #49875

**Reply by Siyanda-m2** [reply]
which will make merging useless since I am only be merging for the benefits, and I'm locked out of the "main" account. isn't there perhaps a way to get the benefits, I don't know perhaps support team do a manual change?

**Reply by davevad93** [reply]
Yes @Siyanda-m2 , in your case it would be useless considering that the account with the benefits is locked. I don't think support can do anything about it, you could try contacting them by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your issue. Although I highly doubt they can do a manual change.
